### DISCUSSION TITLE
Show unresolved suggested changes first

### DISCUSSION THREAD

**Discussion by Pacu2** [community/community#11945] [Pull Requests]
I just had a Code Review with over 50 suggested changes.

The review appeared in the "Conversation" tab of the Pull Request, as seen on the screen.

Only when I clicked on the "35 hidden conversations << Load More >>" button, and then I clicked on another "15 hidden conversations << Load More >>" button, I could see outstanding unresolved comments.

I believe that unresolved comments should be shown first, as otherwise, it's confusing for the reviewers.

Would really appreciate it if we could commit suggestions asynchronously. Right now each commit requires page reload, it might be just the feeling, but the site is loading terribly slow with this amount of suggestions. 

<img width="968" alt="Screenshot 2022-02-22 at 23 33 30" src="https://user-images.githubusercontent.com/21290461/155231096-cc0fab9d-818c-4a4f-a520-8cd6caa2da93.png">

**Comment by Marychoffel22** [comment]
Marychoffel22@smpt.com

**Reply by Marychoffel22** [reply]
Ok waiting for approve

**Reply by Marychoffel22** [reply]
Pull requests

**Comment by Marychoffel22** [comment]
Assigned

**Reply by Marychoffel22** [reply]
Smpt@icloud.com
